R/geom_axes_line.R

```diff
@@ -1,6 +1,6 @@
 #' Add horizontal and/or vertical lines and their legend labels to a plot
 #'
-#' This function is currently designed to be used with ('g_boxplot'), ('g_correlationplot'), ('g_spaghettiplot'),
+#' This function is currently designed to be used with (`'g_boxplot'`), (`'g_correlationplot'`), (`'g_spaghettiplot'`),
```

Comment:
remove `'` signs

Reply:
or even better: convert to markdown references to other documented objects

Reply:
a7530871a6aba2381de651ce1f5287a4196f3609